### PR TITLE
docs: clean up AWS docs after merge

### PR DESCRIPTION
**Pull request by jeremybarbet**
## What

- Merged AWS secrets docs and saw afterward some duplicates old docs

### CODE REVIEW

**Review by eirikurn — APPROVED** [review]
👍🏻

**Review by kksteini — APPROVED** [review]
LGTM

**Review by VilliGunn — APPROVED** [review]
LGTM

**Review by stjanilofts — APPROVED** [review]

File: libs/application/templates/party-letter/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Party Letter
 
-This application allows governmental parties to get assigned a party letter for upcoming elections
+This application allows governmental parties to get assigned a party letter for upcoming elections.
 
 ## Setup
 
-Application System setup: [Setup](https://github.com/island-is/island.is/tree/main/apps/application-system)
+Application System setup: [Setup](https://docs.devland.is/handbook/apps/application-system)
```
Comment (dd-sdang) [inline libs/application/templates/party-letter/README.md:7]: Noticed that the URL is now https://docs.devland.is/apps/application-system (without "handbook" in the path)

**Review by dd-sdang — APPROVED** [review]
Looks good :)

**Pull request merged**